File: frontend/src/api/userModel.js
import BusinessObject from "./BusinessObject";

export default class User extends BusinessObject{

	constructor(aname, aage){
		super();
        this.name = aname
        this.age = aage	
    }

	getName() {
		return this.name;
	}

	setName(aName) {
		this.name = aName;
	}

	getAge() {
		return this.age;
	}

	setAge(aAge) {
		this.age = aAge;
	}

    static fromJSON(user) {
		let results = null;
		if (Array.isArray(user)) {
			results = [];
			user.forEach((x) => {
				Object.setPrototypeOf(x, User.prototype);
				results.push(x);
			})
		} else {
			let x = user;
			Object.setPrototypeOf(x, User.prototype);
			results = x;
		}
		return results;
	}
}

File: backend/model/userModel.py
class User:

    def __init__(self):
        self._id = 0
        self._name = ''
        self._age = ''

    def get_id(self):
        return self._id

    def set_id(self, id):
        self._id = id

    def get_name(self):
        return self._name

    def get_age(self):
        return self._age

    @staticmethod
    def from_dict(dictionary=dict()):
        obj = User()
        obj._id = dictionary['id']
        obj._name = dictionary['name']
        obj._age = dictionary['age']
        return obj
